scripts/python3.8/linux/build.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
```

Comment:
Pretty much all files in this repository are scripts. Adding a "scripts" folder really makes no sense in this regard. Is it not better to put this in the "docker" folder since that's what it's used for and specific to?

Reply:
I also agree on that. We do have a docker/ folder in cura-build-environment, so it's much better to have one in cura-build as well.